.travis.yml

```diff
@@ -1,30 +1,31 @@
-dist: xenial
-sudo: false
 language: python
 cache: pip
-python:
-  - 3.7
-  - 3.6
-  - 3.5
-  - 3.4
-  - 2.7
-before_install:
-  - wget http://download.redis.io/releases/redis-5.0.3.tar.gz && mkdir redis_install && tar -xvzf redis-5.0.3.tar.gz -C redis_install && cd redis_install/redis-5.0.3 && make && src/redis-server --daemonize yes && cd ../..
-  - redis-cli info
-env:
-  - TEST_HIREDIS=0
-  - TEST_HIREDIS=1
-install:
-  - pip install -e .
-  - "if [[ $TEST_PYCODESTYLE == '1' ]]; then pip install pycodestyle; fi"
-  - "if [[ $TEST_HIREDIS == '1' ]]; then pip install hiredis; fi"
-  - "if [[ $TEST_PYCODESTYLE != '1' ]]; then pip install codecov coverage; fi"
-script: "if [[ $TEST_PYCODESTYLE == '1' ]]; then pycodestyle --repeat --show-source --exclude=.venv,.tox,dist,docs,build,*.egg,redis_install .; else coverage run setup.py test; fi"
-after_success:
-  - "if [[ $TEST_PYCODESTYLE != '1' ]]; then codecov; fi"
+services:
+  - redis-server
```

Comment:
While it would be great to rely on Travis's internal redis-server, we want to explicitly control the Redis server vesion. We often build redis-py features against beta versions of the Redis server prior to it being released.